neo4j/result_with_context.go

```diff
@@ -72,15 +75,23 @@ type resultWithContext struct {
 	peekedRecord         *Record
 	peekedSummary        *db.Summary
 	peeked               bool
+	txState              *transactionState
 	afterConsumptionHook func()
 }
 
-func newResultWithContext(connection idb.Connection, stream idb.StreamHandle, cypher string, params map[string]any, afterConsumptionHook func()) ResultWithContext {
+func newResultWithContext(
+	connection idb.Connection,
+	stream idb.StreamHandle,
+	cypher string,
+	params map[string]any,
+	txState *transactionState,
+	afterConsumptionHook func()) ResultWithContext {
```

Comment:
```suggestion
	afterConsumptionHook func(),
) ResultWithContext {
```
Just a personal preference. Otherwise, I find it quite hard to see where the parameter list ends and the function body starts.